src/main/java/ar/com/carloscurotto/storm/complex/topology/propagator/gloss/JMSConfiguration.java

```diff
@@ -0,0 +1,82 @@
+package ar.com.carloscurotto.storm.complex.topology.propagator.gloss;
+
+//import com.jpmorgan.cib.toolkit.util.CtkApplicationUtil;
+
+/**
+ * Holds the different configuration parameters for queue creation.
+ *
+ * @author D540601
+ */
+public class JMSConfiguration {
```

Comment:
Esta clase no es necesaria. La configuracion tiene que estar en Spring y ser inyectada en JMSDestination:

<bean id="mqConnectionFactory" class="com.ibm.mq.jms.MQQueueConnectionFactory">
    <property name="hostName" value="${properties.hostName}" />
    <property name="port" value="${properties.Port}" />
    <property name="queueManager" value="${properties.queuemanager}" />
    // and so on
</bean>